stylus/components/nav.styl

```diff
@@ -91,7 +91,7 @@ $nav-link
         box-shadow   inset 4px 0 0 0 dodgerBlue
         font-weight  bold
 
-        .nav-link__icon
```

Comment:
@GoOz Should we need a retro-compatibility here?

Reply:
Well, ask @ptbrowne I never used it myself yet.

Reply:
Ok, since it was used only for the `<Nav />` component indeed, we don't need one.